src/elements/block/core-embed/embed/iframe/iframe.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { Container, CardMedia, makeStyles } from '@material-ui/core'
+
+const useStyles = makeStyles((theme) => ({
+  cardSize: {
+    height: '300px',
+    margin: theme.spacing(3, 0)
+  }
+}))
+
+type Attributes = {
+  url: string
+  providerNameSlug: string
+}
+
+export type IframeProps = {
+  /**
+   * Container for embed attributes
+   */
+  attributes: Attributes
+}
+
+export function Iframe({ attributes }: IframeProps) {
```

Comment:
```suggestion
export function Iframe({ attributes: { providerNameSlug, url } }: IframeProps) {
```

if you want shorthand access to the params. Otherwise you are repeating attributes each time you are trying to access sub-items.